### PR TITLE
Karam dosa

### PR DESCRIPTION
**Pull request by NeelaReddy25**

### CODE REVIEW

**Comment by NeelaReddy25** [comment]
why you added oil to the karam dosa? already oil is there for plain dosa? why you added specifically

**Comment by NeelaReddy25** [comment]
> _ No description provided. _
extra oil is good for karam dosa

File: menu.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-dosa
\ No newline at end of file
+dosa
+karam
+oil
```
Comment (NeelaReddy25) [inline menu.txt:3]: why oil is added at the end?

Comment (NeelaReddy25) [inline menu.txt:3]: I added at the end. So it may be tasty

**Review by NeelaReddyidamakanti — APPROVED** [review]

**Pull request merged**